purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/trustedentitlements/TrustedEntitlementsInformationalModeIntegrationTest.kt

```diff
@@ -35,20 +36,12 @@ class TrustedEntitlementsInformationalModeIntegrationTest : BasePurchasesIntegra
     }
 
     @Test
-    fun initialCustomerInfoIsVerified() {
-        var receivedCustomerInfo: CustomerInfo? = null
-        ensureBlockFinishes { latch ->
-            Purchases.sharedInstance.getCustomerInfoWith(
-                onError = { fail("should be success. Error: ${it.message}") },
-                onSuccess = {
-                    receivedCustomerInfo = it
-                    latch.countDown()
-                },
-            )
-        }
+    fun initialCustomerInfoIsVerifiedAwaitCustomerInfo() {
```

Comment:
We don't need these "awaits" in the names, that's an implementation detail of how the tests are written now
```suggestion
    fun initialCustomerInfoIsVerified() {
```

Reply:
Great catch! I've removed `AwaitCustomerInfo` from the names when removing the "old" style tests but missed this one 👍

Reply:
Fixed and pushed.